custom-targets/infrastructure-manager/quickstart/QUICKSTART.md

```diff
@@ -0,0 +1,191 @@
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/config.agent"
+    ```
+
+4. `config.admin` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/config.admin"
+    ```
+
+5. `compute.networkAdmin` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/compute.networkAdmin"
+    ```
+
+## 4. Build and Register a Custom Target Type for Infrastructure Manager
+From within the `quickstart` directory, run the following command to build the Infrastructure Manager deployer image and register a Cloud Deploy custom target type that references the image:
+
+```shell
+../build_and_register.sh -p $PROJECT_ID -r $REGION
+```
+
+For information about the `build_and_register.sh` script, see the [README](../README.md#build)
+
+## 5. Create delivery pipeline and targets
+Replace the placeholders in the `clouddeploy.yaml` in this directory with your environment variable values:
+
+```shell
+sed -i "s/\$PROJECT_ID/${PROJECT_ID}/g" ./clouddeploy.yaml
+sed -i "s/\$REGION/${REGION}/g" ./clouddeploy.yaml
+```
+
+Apply the Cloud Deploy configuration:
+
+```shell
+gcloud deploy apply --file=clouddeploy.yaml --project=$PROJECT_ID --region=$REGION
+```
+
+## 6. Create a release
+Create a Cloud Deploy release for the configuration defined in the `configuration` directory. This will automatically create a rollout that deploys the Terraform configuration to the dev target.
+
+```shell
+gcloud deploy releases create release-001 --delivery-pipeline=im-network-pipeline --source=configuration --project=$PROJECT_ID --region=$REGION
+```
+
+### Configuration context
+The Terraform configuration is structured so the dev and prod root modules are defined in their own directories:
+
+* `configuration/dev`
+* `configuration/prod`
+
+The path to the relevant configuration is provided as a deploy parameter on the delivery pipeline stage. Additionaly, the configuration expects `project_id` variable to be set, this is provided as a deploy parameter on the targets.
+
+## 7. Check rollout status for dev target
+In the [Cloud Deploy UI](https://console.cloud.google.com/deploy/delivery-pipelines) for your project click on the `im-network-pipeline` delivery pipeline. Here you can see the release created and the rollout to the dev target for the release.
+
+You can also describe the rollout created using the following command:
+
+```shell
+gcloud deploy rollouts describe release-001-to-im-dev-0001 --release=release-001 --delivery-pipeline=im-network-pipeline --project=$PROJECT_ID --region=$REGION
```

Comment:
Consider explaining how to find the latest release/rollout, its not uncommon to make a mistake during a quickstart and needing to trigger another rollout